LayoutTests/fast/inline/legacy-margin-trim/block-container-float-multiple-formatting-contexts-inline-layout.html

```diff
@@ -0,0 +1,36 @@
+<!-- webkit-test-runner [ InlineFormattingContextIntegrationEnabled=false ] -->
+<!DOCTYPE html>
+<html>
+<head>
+<link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com">
+<link rel="help" href="https://drafts.csswg.org/css-box-4/#margin-trim-block">
+<style>
+container {
+    margin-trim: block;
+    display: flow-root;
+    width: max-content;
+}
+item {
+    display: block;
+    width: 25px;
+    height: 100px;
+    margin-block-end: 50px;
+    float: left;
+}
+span {
+    display: inline-block;
```

Comment:
this is rather misleading. what's the point of making a <span> and turn it into an inline-block?

Reply:
Yeah not sure why I did that. Should be changed now